innopoints/schemas/account.py

```diff
@@ -30,3 +32,15 @@ class Meta:
     entry_time = ma.DateTime(timezone=True)
     type = ma.Str()
     payload = ma.Dict()
+
+
+class NotificationSettingsSchema(ma.Schema):
+    class Meta:
+        ordered = True
+        sqla_session = db.session
+
+    innostore = ma.Str(validate=validate.OneOf(['off', 'email', 'push']))
+    volunteering = ma.Str(validate=validate.OneOf(['off', 'email', 'push']))
+    project_creation = ma.Str(validate=validate.OneOf(['off', 'email', 'push']))
+    administration = ma.Str(validate=validate.OneOf(['off', 'email', 'push']))
+    service = ma.Str(validate=validate.OneOf(['off', 'email', 'push']))
```

Comment:
A lot of redundancy. factor the list into an `allowed_modes` or something so that any edit to them would be in one place.